src/CMakeLists.txt

```diff
@@ -170,11 +170,21 @@ else() # Other compiler/version are unsupported, use external lib
 endif()
 
 ## stb ##
-execute_process(
-	COMMAND git submodule update --init src/stb
-	WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/stb)
+if(NOT WIN32 OR MINGW)
+	pkg_check_modules(STB REQUIRED stb)
+endif()
```

Comment:
Maybe we should make `pkg_check_modules()` a no-op instead of performing a check every time we use it.

Alternatively: https://github.com/mumble-voip/mumble/blob/8fc359ea3fafbbb136abece6d422af409274b23e/cmake/pkg-utils.cmake